src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceSubmittedCallbackHandler.java

```diff
@@ -72,31 +76,41 @@ private boolean isFurtherEvidenceActionOptionValid(DynamicList furtherEvidenceAc
 
     private SscsCaseDetails updateCase(Callback<SscsCaseData> callback, SscsCaseData caseData) {
         if (isFurtherEvidenceActionOptionValid(caseData.getFurtherEvidenceAction(),
-            INFORMATION_RECEIVED_FOR_INTERLOC_JUDGE)) {
+                INFORMATION_RECEIVED_FOR_INTERLOC_JUDGE)) {
             caseData.setInterlocReferralDate(LocalDate.now().toString());
             return setInterlocReviewStateFieldAndTriggerEvent(caseData, callback.getCaseDetails().getId(),
                     REVIEW_BY_JUDGE.getId(), INFORMATION_RECEIVED_FOR_INTERLOC_JUDGE,
-                EventType.INTERLOC_INFORMATION_RECEIVED_ACTION_FURTHER_EVIDENCE, "Interloc information received event");
+                    EventType.INTERLOC_INFORMATION_RECEIVED_ACTION_FURTHER_EVIDENCE, "Interloc information received event");
         }
         if (isFurtherEvidenceActionOptionValid(caseData.getFurtherEvidenceAction(),
-            INFORMATION_RECEIVED_FOR_INTERLOC_TCW)) {
+                INFORMATION_RECEIVED_FOR_INTERLOC_TCW)) {
             caseData.setInterlocReferralDate(LocalDate.now().toString());
             return setInterlocReviewStateFieldAndTriggerEvent(caseData, callback.getCaseDetails().getId(),
-                REVIEW_BY_TCW.getId(), INFORMATION_RECEIVED_FOR_INTERLOC_TCW,
-                EventType.INTERLOC_INFORMATION_RECEIVED_ACTION_FURTHER_EVIDENCE, "Interloc information received event");
+                    REVIEW_BY_TCW.getId(), INFORMATION_RECEIVED_FOR_INTERLOC_TCW,
+                    EventType.INTERLOC_INFORMATION_RECEIVED_ACTION_FURTHER_EVIDENCE, "Interloc information received event");
         }
         if (isFurtherEvidenceActionOptionValid(caseData.getFurtherEvidenceAction(),
-            SEND_TO_INTERLOC_REVIEW_BY_JUDGE)) {
+                SEND_TO_INTERLOC_REVIEW_BY_JUDGE)) {
             setSelectWhoReviewsCaseField(caseData, REVIEW_BY_JUDGE);
             return setInterlocReviewStateFieldAndTriggerEvent(caseData, callback.getCaseDetails().getId(),
-                REVIEW_BY_JUDGE.getId(), SEND_TO_INTERLOC_REVIEW_BY_JUDGE,
-                EventType.VALID_SEND_TO_INTERLOC, "Send a case to a judge for review");
+                    REVIEW_BY_JUDGE.getId(), SEND_TO_INTERLOC_REVIEW_BY_JUDGE,
+                    EventType.VALID_SEND_TO_INTERLOC, "Send a case to a judge for review");
```

Comment:
Could replace the string here with the variable you created 'TCW_REVIEW_SEND_TO_JUDGE'

Reply:
sorted